src/App.js

```diff
@@ -1,24 +1,14 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import Title from "./Components/Title/Title.mondule";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Title />}></Route>
+      </Routes>
+    </BrowserRouter>
```

Comment:
No need to change anything here, but we'll be using the new routers from v6.4 later since the react-router folks [don't recommend we use `BrowserRouter` anymore](https://reactrouter.com/en/main/routers/picking-a-router#using-v64-data-apis).